srfi-64.html

```diff
@@ -68,7 +68,7 @@ <h2>Rationale</h2>
 and terse to use, but it may be a little less elegant and <q>compositional</q>
 than using explicit test objects, such as JUnit-style frameworks.
 It is not claimed to follow either object-oriented or functional design
-principles, but I hope it is useful and convenient to use and extend.</p>
+principles, but it is hoped to be useful and convenient to use and extend.</p>
```

Comment:
Using the passive voice is generally viewed as less good.  But if Per agrees, I'd be happy to make it.

Reply:
Oh? I'm used to specifications and scientific publications more generally being authored in passive voice, so I'm surprised by that.

Reply:
> Oh? I'm used to specifications and scientific publications more generally being authored in passive voice, so I'm surprised by that.

Yes, it happens a lot, but is frowned upon.  All the standard guides to good English style, e.g. Strunk and White, recommend using the active voice wherever possible.  If you search "active voice" on Google, you'll find many  reputable sites saying that.  In this example, are you trying to avoid the use of the first-person pronoun?  Academic writing often does avoid that.  Still, given that SRFI 64 is almost twenty years old, I don't want to change it unless, like in the other two cases, there is an actual error.

Reply:
Yes, I was referring to academic writing; I  was taught to avoid using 'we' or 'I' in scientific reports and articles, and specs being very formal typically, seems a good place to avoid active voice too, or so I thought.

Reply:
I've now dropped the passive voice change.